app/js/actions/createLabOrder.js

```diff
@@ -0,0 +1,185 @@
+          provider: 'f9badd80-ab76-11e2-9e96-0800200c9a66',
+        },
+      ],
+      encounterType: '7b68d557-85ef-4fc8-b767-4fa4f5eb5c23',
+      location: {
+        uuid: 'b1a8b05e-3542-4037-bbd3-998ee9c40574',
+        display: 'Inpatient Ward',
+        name: 'Inpatient Ward',
+        description: null,
+        address1: null,
+        address2: null,
+        cityVillage: null,
+        stateProvince: null,
+        country: null,
+        postalCode: null,
+        latitude: null,
+        longitude: null,
+        countyDistrict: null,
+        address3: null,
+        address4: null,
+        address5: null,
+        address6: null,
+        tags: [
+          {
+            uuid: '8d4626ca-7abd-42ad-be48-56767bbcf272',
+            display: 'Transfer Location',
+            links: [
+              {
+                rel: 'self',
+                uri:
+                  'http://localhost:8081/openmrs-standalone/ws/rest/v1/locationtag/8d4626ca-7abd-42ad-be48-56767bbcf272',
+              },
+            ],
+          },
+          {
+            uuid: 'b8bbf83e-645f-451f-8efe-a0db56f09676',
+            display: 'Login Location',
+            links: [
+              {
+                rel: 'self',
+                uri:
+                  'http://localhost:8081/openmrs-standalone/ws/rest/v1/locationtag/b8bbf83e-645f-451f-8efe-a0db56f09676',
+              },
+            ],
+          },
+          {
+            uuid: '1c783dca-fd54-4ea8-a0fc-2875374e9cb6',
+            display: 'Admission Location',
+            links: [
+              {
+                rel: 'self',
+                uri:
+                  'http://localhost:8081/openmrs-standalone/ws/rest/v1/locationtag/1c783dca-fd54-4ea8-a0fc-2875374e9cb6',
+              },
+            ],
+          },
+        ],
+        parentLocation: {
+          uuid: 'aff27d58-a15c-49a6-9beb-d30dcfc0c66e',
+          display: 'Amani Hospital',
+          links: [
+            {
+              rel: 'self',
+              uri:
+                'http://localhost:8081/openmrs-standalone/ws/rest/v1/location/aff27d58-a15c-49a6-9beb-d30dcfc0c66e',
+            },
+          ],
+        },
+        childLocations: [],
+        retired: false,
+        attributes: [],
+        address7: null,
+        address8: null,
+        address9: null,
+        address10: null,
+        address11: null,
+        address12: null,
+        address13: null,
+        address14: null,
+        address15: null,
+        links: [
+          {
+            rel: 'self',
+            uri:
+              'http://localhost:8081/openmrs-standalone/ws/rest/v1/location/b1a8b05e-3542-4037-bbd3-998ee9c40574',
+          },
+          {
+            rel: 'full',
+            uri:
+              'http://localhost:8081/openmrs-standalone/ws/rest/v1/location/b1a8b05e-3542-4037-bbd3-998ee9c40574?v=full',
+          },
+        ],
+        resourceVersion: '2.0',
+      },
+      orders: [
```

Comment:
I think I see the issue here... test drugs have different parameters that drug orders... let me collect what they are and will post on the Talk post.